src/server/sv_entity.c

```diff
@@ -177,7 +177,7 @@ static void Sv_ClientVisibility(const vec3_t org, byte *pvs, byte *phs) {
 		byte cluster_pvs[MAX_BSP_LEAFS >> 3];
 		Cm_ClusterPVS(clusters[i], cluster_pvs);
 
-		for (size_t i = 0; i < sizeof(pvs); i++) {
+		for (size_t i = 0; i < MAX_BSP_LEAFS >> 3; i++) {
 			pvs[i] |= cluster_pvs[i];
 		}
 	}
```

Comment:
sizeof(*byte) = 8 on 64 bit systems and 4 on 32 bit systems (I think)
I've just tested this on my system and sizeof(pvs) was 8.